internal/driver/driver.go

```diff
@@ -155,6 +163,18 @@ func generateReport(p *profile.Profile, cmd []string, cfg config, o *plugin.Opti
 	return out.Close()
 }
 
+func weblist(dst io.Writer, rpt *report.Report, obj plugin.ObjTool) error {
```

Comment:
Can you rename this method to start with a verb as well to be consistent with the rest of the method? Maybe just the old printWebList can work, as it does print to the input io writer.